src/main/java/com/zaxxer/hikari/pool/PoolBase.java

```diff
@@ -445,9 +439,6 @@ private int getAndSetNetworkTimeout(final Connection connection, final long time
                if (validationTimeout < SECONDS.toMillis(1)) {
                   LOGGER.warn("{} - A validationTimeout of less than 1 second cannot be honored on drivers without setNetworkTimeout() support.", poolName);
                }
-               else if (validationTimeout % SECONDS.toMillis(1) != 0) {
-                  LOGGER.warn("{} - A validationTimeout with fractional second granularity cannot be honored on drivers without setNetworkTimeout() support.", poolName);
-               }
```

Comment:
Why was this removed?  This is intended to log a warning in cases where the user configures a ``validationTimeout`` with fractional second granularity, for example, ``2250ms`` but does not have a driver that supports ``setNetworkTimeout()``.